x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/text_expansion_callout_logic.test.ts

```diff
@@ -293,6 +299,30 @@ describe('TextExpansionCalloutLogic', () => {
       });
     });
 
+    describe('textExpansionError', () => {
+      const error = {
+        body: {
+          error: 'Error with ELSER deployment',
+          message: 'Mocked error message',
+          statusCode: 500,
+        },
+      } as HttpError;
+
+      it('returns null when there are no errors', () => {
+        CreateTextExpansionModelApiLogic.actions.apiReset();
+        FetchTextExpansionModelApiLogic.actions.apiReset();
+        StartTextExpansionModelApiLogic.actions.apiReset();
+        expect(TextExpansionCalloutLogic.values.textExpansionError).toBe(null);
+      });
+      it('returns extracted error for create', () => {
```

Comment:
You should write tests for each of the 3 possible errors.

Reply:
Good catch!! I added unit tests for other errors. Thank you.